lib/controller/page/type/page.summary/page.summary.controller.unit.spec.js

```diff
@@ -18,11 +18,17 @@ const submitterClientSpy = spy(submitterClient, 'submit')
 
 const checkSubmitsStub = (_instanceData, _userData) => true
 
+const pdfPayload = {some: 'PDF contents'}
+const pdfPayloadStub = (_rawFormData) => pdfPayload
+const submissionDataWithLabelsStub = stub()
+
 const pageSummaryController = proxyquire('./page.summary.controller', {
   '../../../../route/route': route,
   '../../../../service-data/service-data': serviceData,
   '../../../../page/redirect-next-page/redirect-next-page': redirectNextPageStub,
-  '../../../../client/submitter/submitter': {submitterClient: submitterClientSpy}
+  '../../../../client/submitter/submitter': {submitterClient: submitterClientSpy},
+  '../../../../presenter/pdf-payload': {pdfPayload: pdfPayloadStub},
+  '../../../../middleware/routes-output/submission-data-with-labels': {submissionDataWithLabels: submissionDataWithLabelsStub}
```

Comment:
So much churn with this test setup.
Unable to reduce this, leaving for now.  Test can do with a refactor.